url/urltestdata.json

```diff
@@ -4587,5 +4587,76 @@
     "pathname": "/",
     "search": "",
     "hash": ""
+  },
+  "# More file URL tests by zcorpan",
```

Comment:
I just wanted to separate these tests from the ones above to direct any blame at me, but of course `git blame` works too. Can remove it though.